infra-core/pom.xml

```diff
@@ -10,7 +10,7 @@
   </parent>
 
   <modelVersion>4.0.0</modelVersion>
-  <artifactId>infra</artifactId>
+  <artifactId>infra-core</artifactId>
```

Comment:
what about creating an `infra` packaging-pom with 3 submodules `core`, `v1` and `v2`?